jbmc/regression/jbmc-strings/ConstantEvaluationContains/test.desc

```diff
@@ -0,0 +1,8 @@
+CORE
+Main.class
+--function Main.contains --show-vcc --cp `../../../../scripts/format_classpath.sh . ../../../lib/java-models-library/target/core-models.jar`
+^Generated [0-9]+ VCC\(s\), 0 remaining after simplification$
```

Comment:
Since `--show-vcc` is used, the test only prints the VCCs and checks that we have 0 remaining VCCs after symex. We also need to check though that constant propagation does compute the correct result. We should leave off `--show-vcc` and check that verification succeeded. (also for the other tests)

Also we should check the negative cases, i.e. here if a string doesn't contain another string.